### PR TITLE
Prims algorithm using c++ stl

### PR DESCRIPTION
**Pull request by sg28march**

### CODE REVIEW

**Comment by sg28march** [comment]
prims algorithm #101

File: Algorithms/Graphs/primsAlgo.cpp [hunk 1]
```diff
@@ -0,0 +1,119 @@
+
```
Comment (Bhavnaharitsa) [inline Algorithms/Graphs/primsAlgo.cpp:1]: add algorithm name, description, sample input output

File: Algorithms/Graphs/primsAlgo.cpp [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+# define INF 0x3f3f3f3f
+ 
+// iPair ==>  Integer Pair
+typedef pair<int, int> iPair;
+ 
+// This class represents a directed graph using
+// adjacency list representation
+class Graph
+{
+    int V;    // No. of vertices
```
Comment (Bhavnaharitsa) [inline Algorithms/Graphs/primsAlgo.cpp:13]: no single letter variable

File: Algorithms/Graphs/primsAlgo.cpp [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+// Allocates memory for adjacency list
+Graph::Graph(int V)
+{
+    this->V = V;
+    adj = new list<iPair> [V];
+}
+ 
+void Graph::addEdge(int u, int v, int w)
+{
+    adj[u].push_back(make_pair(v, w));
```
Comment (Bhavnaharitsa) [inline Algorithms/Graphs/primsAlgo.cpp:37]: no single letter variable

File: Algorithms/Graphs/primsAlgo.cpp [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+ 
+// Driver program to test methods of graph class
+int main()
+{
+    // create the graph given in above fugure
+    int V = 9;
+    Graph g(V);
+ 
+    //  making above shown graph
+    g.addEdge(0, 1, 4);
```
Comment (Bhavnaharitsa) [inline Algorithms/Graphs/primsAlgo.cpp:103]: make it dynamic not static

File: Algorithms/Graphs/primsAlgo.cpp [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    g.addEdge(0, 7, 8);
```
Comment (Bhavnaharitsa) [inline Algorithms/Graphs/primsAlgo.cpp:104]: g no single letter variable

**Pull request closed without merging**